src/main/java/com/uber/profiling/YamlConfigProvider.java

```diff
@@ -138,13 +138,7 @@ public void setFilePath(String filePath) {
                         }
                         
                         Map<Object, Object> overrideValues = (Map<Object, Object>)valueObj;
-                        for (Map.Entry<Object, Object> entry : overrideValues.entrySet()) {
-                            if (entry.getValue() == null) {
-                                continue;
-                            }
-                            String configKey = entry.getKey().toString();
-                            addConfig(result, overrideKey, configKey, entry.getValue());
-                        }
+                        addConfig(result, overrideKey, OVERRIDE_KEY + "." + overrideKey, overrideValues);
```

Comment:
We do not want to put OVERRIDE_KEY in the config name. 
For example,
```
{
key1: value1,
override:
  override1:
     key1:value2
}
```
We want to generate config like "key1->value2" instead of "override.override1.key1->value2".

Reply:
Thanks @boy-uber, that makes sense. I committed the modified files.